modules/system/classes/PluginManager.php

```diff
@@ -398,12 +396,7 @@ public function bootPlugin(PluginBase $plugin): void
      */
     public function getPluginPath(string|PluginBase $plugin): ?string
     {
-        $path = null;
-        $plugin = $this->findByIdentifier($plugin, true);
-        if ($plugin) {
-            $path = $plugin->getPluginPath();
-        }
-        return $path;
+        return $this->findByIdentifier($plugin, true)?->getPluginPath();
```

Comment:
Hmm, not sure I'm a fan of this syntax, could be easily missed. @bennothommo any thoughts?

Reply:
I have used it before (not in core, in my own projects). It can be easy to miss, but it does take out the need for a lot of conditionals and is great for fluent/chained calls.